snap/snapcraft.yaml

```diff
@@ -0,0 +1,267 @@
+name: ubuntu-desktop-installer
```

Comment:
I've kept the name `ubuntu-desktop-installer` for now. Do we plan to keep it this way, or should we use `ubuntu-bootstrap` in the future to avoid confusion (since it's supposed to be able to install more than just ubuntu-desktop)?